xml/System.Text.Encodings.Web/HtmlEncoder.xml

```diff
@@ -52,10 +52,10 @@
         <Parameter Name="settings" Type="System.Text.Encodings.Web.TextEncoderSettings" />
       </Parameters>
       <Docs>
-        <param name="settings">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="settings">Settings used to control how the created <see cref="HtmlEncoder"/> encodes, primarily which characters to encode.</param>
```

Comment:
nit: used to control --> that control
nit: how the created HTMLEncoder encodes --> how the HtmlEncoder instance encodes
with provided -> with the provided